scripts/haproxy/s3haproxyconfig/s3haproxyconfig/s3_haproxy_config.py

```diff
@@ -113,11 +122,25 @@ def run():
     # option log-health-checks
     default-server inter 2s fastinter 100 rise 1 fall 5 on-error fastinter
 
-    server s3authserver-instance1 ''')
-    target.write(pvt_ip)
-    target.write(''':28050 #check ssl verify required ca-file /etc/ssl/stx-s3/s3auth/s3authserver.crt   # s3 auth server instance 1
-#----------------------------------------------------------------------
-''')
+'''
+
+    str3 = '''
+'''
+
+    target.write(header_text)
+    target.write(str1)
+    for i in range(0, n):
+        target.write(
+        "    server s3-instance-%s %s:%s check maxconn 110        # s3 instance %s\n"
+        % (i+1, pvt_ip, s3inport+i, i+1))
+    target.write(str2)
+    for i in range(0, n):
```

Comment:
Auth can remain only one instance. No need to have a loop here

Reply:
Done